gym_extensions/continuous/mujoco/__init__.py

```diff
@@ -100,11 +119,26 @@
                 dict(path='gym_extensions.continuous.mujoco.modified_humanoid:HumanoidGravityEnv',
                      max_episode_steps=1000,
                      kwargs= dict(gravity=-12.2625)),
-            "HopperWall-v0" :
-                dict(path='gym_extensions.continuous.mujoco.modified_hopper:HopperWallEnv',
+
+
+
+            ### Environment with walls
+            "AntMaze-v0" :
+                dict(path='gym_extensions.continuous.mujoco.modified_ant:AntMaze',
                      max_episode_steps=1000,
                      reward_threshold=3800.0,
                      kwargs= dict()),
+            "HopperStairs-v0" :
+                dict(path='gym_extensions.continuous.mujoco.modified_hopper:HopperStairs',
+                     max_episode_steps=1000,
+                     reward_threshold=3800.0,
+                     kwargs= dict()),
+            "HopperIce-v0" :
```

Comment:
I think we should change this to HopperWallIceEnv-v0 -> The point is the wall is icy not the whole floor right?